README.md

```diff
@@ -159,46 +72,84 @@ The summary details will be accessible in `$RUNNER_TEMP/build-scan-data-maven/bu
       "requestedTasks": "install",
       "buildId": "1701252760831-89e3583d-3c3c-4a64-a958-e61aa791f4f4",
       "buildFailure": true,
+      "buildTimestamp": "1701252760800",
       "buildScanLink": "https://<DEVELOCITY_URL>/s/dxqnlj5hgybq4"
     }
   ]
 }
 ```
-By default, a summary will be added to the GitHub workflow calling the action (can be skipped with `skip-job-summary` is set to `true`):
+#### Capture unpublished Build Scan®
 
-![workflow](./doc/summary-workflow.png)
+The _capture strategy_ can be customized:
+- `ALWAYS`: default behavior, capture will be attempted on each Maven invocation
```

Comment:
I think we discussed this before, but the next Develocity maven extension will not feature these values like `ALWAYS`, `ON_FAILURE`, and `ON_DEMAND`. It will only have conditional publication. `onlyIf`. Can you remind me why we didn't align this API?

Reply:
as far as I remember, I didn't want to execute arbitrary code in the Maven process. That would also force to ship a parser / interpreter.

Reply:
got it, thanks